src/widgets/Chart/model/chart.store.ts

```diff
@@ -0,0 +1,15 @@
+import { makeAutoObservable } from 'mobx';
+import { type ExpenseInterface } from '../interfaces/expense.interface';
+import { expensesApi } from '../api/chart.api';
+
+export class ChartStore {
+  chartData: ExpenseInterface[] = [];
```

Comment:
for future reference every essential value must be private, we can't just mutate or retrieve this data from the class without the function for it